includes/fraud-prevention/class-buyer-fingerprinting-service.php

```diff
@@ -54,15 +54,33 @@ public function hash_data_for_fraud_prevention( string $data ): string {
 		return hash( 'sha512', $data, false );
 	}
 
+	/**
+	 * Returns the 3 octet format from an IP address.
+	 *
+	 * @param string $ip_address Raw IP address.
+	 *
+	 * @return string 3 octet IP address (192.192.192).
+	 */
+	public function get_3_octet_from_ip( string $ip_address ): string {
+		$ip_address_octets = explode( '.', $ip_address );
```

Comment:
Is there a specific reason why we are not taking into account IPv6 here?

Reply:
Thanks for your comment @kalessil. In fact, we didn't consider IPv6 while thinking about this solution. I'll adapt the nomenclatures and the code to be able to handle both versions.

Reply:
Updated on [084a732](https://github.com/Automattic/woocommerce-payments/pull/4709/commits/084a7328577a9070bf78680127a3644d27d3e385) and [1d3e464](https://github.com/Automattic/woocommerce-payments/pull/4709/commits/1d3e4647ffaa5d864fda90aba3e214604605f4a0).